### DISCUSSION TITLE
NSFW warning for profile image, background images and links

### DISCUSSION THREAD

**Discussion by trymeouteh** [JulianPrieber/littlelink-custom#674] [Feature requests]
Please add a setting to check off weather a profile image is NSFW,  background image is NSFW which will prompt the user with a message on weather to view the background image or not.

Also add a NSFW checkbox next to every link which will show a NSFW icon next to them. When a user clicks on a NSFW link, they will be warned before proceeding.

This is useful for instances who want to host NSFW content but have it blocked by default unless the user alllows it. This will prevent any "unwanted surprises" such as seeing a nude background image or click on a link to an adult site.

**Comment by Hidden-Warden** [comment]
This indeed would be a great addition ! Adding an NSFW option for profile and background images would be a great way to help users control the content they see. This would provide an extra layer of control for users who want to avoid unexpected or unwanted content. 

Definitely a useful feature!

**Comment by Baralheia** [comment]
+1 voice in support... I'm really liking LinkStack, but having the ability to mark a link as "Adult Content"/"NSFW" so that it shows users a warning popup/page when clicking on it is one feature I miss from Linktree.
